### DISCUSSION TITLE
couldn't reactivate SD card

### DISCUSSION THREAD

**Discussion by thutatheinnaing** [2bllw8/anemo#62] [Q&A]
I clicked this icon beside SD card and now I couldn't find an option to reactivate my SD card!

![Screenshot_2024-11-03-02-36-04-967_com google android documentsui](https://github.com/user-attachments/assets/9b22f8fc-9c60-4aad-9c3e-3f97ee9a4b8e)

**Comment by nain-F49FF806** [comment]
Restart your device. It should work. 

If it doesn't. Remove the SD card and re-insert.
